src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java

```diff
@@ -420,6 +421,9 @@ public ResponseEntity<CCDCallbackResponse> amendRespondentDetails(
         List<String> errors = eventValidationService.validateActiveRespondents(caseData);
         if (errors.isEmpty()) {
             errors = eventValidationService.validateET3ResponseFields(caseData);
+            if(errors.isEmpty()) {
+                errors = InvalidCharacterCheck.checkNamesForInvalidCharacters(caseData, "respondent");
```

Comment:
I think this also needs to be added in the aboutToStart for the ET3 Processing event.